editoast/openapi.yaml

```
@@ -1324,6 +1324,39 @@ paths:
                         train_path:
                           type: number
 
+  /timetable/{id}/conflicts/:
+    get:
+      tags:
+        - timetable
+      summary: Retrieve all conflicts for a specific timetable
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: Timetable ID
+          required: true
+      responses:
+        200:
+          description: The timetable conflicts content
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    trains_id:
```

```suggestion
                    train_ids:
```
Here you have trains_id, but in Conflict, you have train_ids :)

---

fixed !